clients/java/src/test/java/io/zeebe/client/ZeebeClientTest.java

```diff
@@ -47,4 +52,19 @@ public void shouldHaveDefaultValues() {
       assertThat(configuration.getDefaultRequestTimeout()).isEqualTo(Duration.ofSeconds(20));
     }
   }
+
+  @Test
+  public void shouldFailIfCertificateDoesNotExist() {
+    // given
+    thrown.expectCause(IsInstanceOf.instanceOf(FileNotFoundException.class));
+
+    // when
+    ZeebeClient.newClientBuilder().useSecureConnection().caCertificatePath("/wrong/path").build();
+  }
+
+  @Test
+  public void shouldAllowEmptyCertificatePath() {
```

Comment:
I think if the user explicitly passes something to `caCertificatePath` then we should validate it, no?

Reply:
Do you mean in the "" case or something else?